color.main.test.js

```diff
@@ -0,0 +1,149 @@
+/**
+ *@jest-environment jsdom
+ */
+
+ describe ('check out change text color after the click', () => {
+     let colors = ['magenta', 'cyan', 'firebrick', 'springgreen', 'skyblue'];
+
+     document.body.innerHTML = `
+     <p id="text1">text</p>
+     <p id="text2">text</p>
+     <p id="text3">text</p> 
+    `
+    let text1 = document.getElementById('text1');
+    let text2 = document.getElementById('text2');
+    let text3 = document.getElementById('text3');
+     
+    let k=0;
+    let j=0;
+    let z=0;
```

Comment:
Better import your functions cause potentially you can change you main file and (f.e) broke logic, but your tests will passed